src/platform/Darwin/DnssdContexts.cpp

```diff
@@ -458,27 +458,35 @@ ResolveContext::ResolveContext(void * cbContext, DnssdResolveCallback cb, chip::
                                std::shared_ptr<uint32_t> && consumerCounterToUse) :
     browseThatCausedResolve(browseCausingResolve)
 {
-    type            = ContextType::Resolve;
-    context         = cbContext;
-    callback        = cb;
-    protocol        = GetProtocol(cbAddressType);
-    instanceName    = instanceNameToResolve;
-    consumerCounter = std::move(consumerCounterToUse);
+    type               = ContextType::Resolve;
+    context            = cbContext;
+    callback           = cb;
+    protocol           = GetProtocol(cbAddressType);
+    instanceName       = instanceNameToResolve;
+    consumerCounter    = std::move(consumerCounterToUse);
+    hasSrpTimerStarted = false;
```

Comment:
Why is this not default-initialized in the class decl, so you don't have to do it in both constructors?

I know other members were doing things this way; that's not a reason to copy the bad pattern.

Reply:
> i already have it defaulting to false in the class decl.

Then we shouldn't touch it in the constructor.